src/actions/update.js

```diff
@@ -31,29 +66,47 @@ const updateRepository = async (adapter, repo, packageName, packageVersion, opti
     packageDef.dependencies[packageName] = packageVersion
   }
 
+  commitUpdates(ACTION, adapter, options, repository, branchName, packageDef, packages.lockfile.create)
+}
+
+const updateRepositoryAll = async (adapter, repository, options) => {
+  notify(ALL_ACTION, `Running npm update against ${chalk.italic(repository.fullName)}`)
+  const branchName = `turnup/update-all`
```

Comment:
To make sure the user can run this again and not encounter an issue with existing branch name, let's add some kind of unique value here. Perhaps bring in the `uuid` library and use `v4()`, slicing off the first six characters.

Reply:
And appending to `turnup/update-all`.

Reply:
Good call e45c662ab012952d851ab5b221752215ce1272af